jekyll/_cci2/docker-layer-caching.md

```diff
@@ -17,20 +17,19 @@ Consider reusing the unchanged layers to significantly reduce image build times.
 
 ``` YAML
 - setup_remote_docker:
-    reusable: true    # default - false
-    exclusive: true   # default - true
+    docker_layer_caching: true # default - true  
 ```
 
-## **`reusable`**
+When `docker_layer_caching` is set to `true`, CircleCI will try to reuse your Remote Docker Environment. That is, every layer you built in a previous job will be accessible in the remote environment and CircleCI will attempt to reuse the previous environment when it is possible. However, in some cases your job may run in a clean environment, even if the configuration specifies `docker_layer_caching: true`.
```

Comment:
I would rather say something like "CircleCI will try to reuse Docker Images (layers) from your previous builds"  instead of "CircleCI will try to reuse your Remote Docker Environment"

Reply:
"and CircleCI will attempt to reuse the previous environment when it is possible" seems like a repetition and I would probably avoid using phrases like "reuse environment" because it might give wrong expectations.

Reply:
great, I made the first change and removed the second.